exdb/views.py

```diff
@@ -108,18 +106,22 @@ def form_valid(self, form):
         form.instance.experience = self.get_experience()
         form.instance.experience.status = 'ad' if self.request.POST.get('approve') else 'de'
         if form.instance.experience.status == 'ad':
-            form.instance.experience.approver = self.request.user
-            form.instance.experience.approved_timestamp = now()
+            form.instance.experience.current_approver = self.request.user
```

Comment:
We should add some security in here to check if the `current_approver` is the one trying to approve it instead of blindly setting it.